observatory-platform/observatory/platform/utils/test_utils.py

```diff
@@ -121,14 +121,15 @@
 from pendulum import DateTime
 from sftpserver.stub_sftp import StubServer, StubSFTPServer
 import freezegun
+import socket
 
 
 def random_id():
     """Generate a random id for bucket name.
 
     :return: a random string id.
     """
-    return str(uuid.uuid4()).replace("-", "")
+    return str(uuid.uuid5(uuid.uuid4(), socket.gethostname())).replace("-", "")
```

Comment:
It would be good to leave a comment about why we are doing this in the code, for future reference, for example:

When code is pushed to a branch and a pull request is open, Github Actions runs the unit tests workflow twice, one for the push and one for the pull request. However, the `uuid4` function, which calls `os.urandom(16)`, generates the same sequence of values for each workflow run. We have also used the hostname of the machine in the construction of the random id, to make sure that the random ids are different on both workflow runs.